orc/src/java/org/apache/orc/DataReader.java

```diff
@@ -53,4 +62,9 @@ DiskRangeList readFileData(
    * @param toRelease The buffer to release.
    */
   void releaseBuffer(ByteBuffer toRelease);
+
+  DataReader clone();
```

Comment:
Does the clone the options/properties alone? or the fs input stream as well? Can you please add a comment as in what does it clone and what not.